SU-HSDO/suhumsci
# fix(STN-434): timing issue with .hb-stretch-vertical-linked-cards

# READY FOR REVIEW

## Summary
Implements both a timeout and promise behavior to try to ensure the stretch cards load correctly. This may still not fix the issue on extremely slow connections.

## Steps to Test
1. `npm run test` passes
2. see the Jira issue here: https://sparkbox.atlassian.net/browse/STN-434 - watch the video and see if you can still replicate the bug!

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
